src/Cardano/Wallet/Primitive.hs

```diff
@@ -117,6 +124,15 @@ data BlockHeader = BlockHeader
 
 instance NFData BlockHeader
 
+instance Buildable BlockHeader where
+    build (BlockHeader s prev) = mempty
```

Comment:
Do we need `mempty` here?

Reply:
Not really. But it makes the alignment nicer below. With monoids or applicatives, I usually like starting with an empty case and then, on each newline, have a consistent alignment with `<>  ...`, rather than having to do something like:

```hs
fn =
       myFirstCase
    <> MySecondCase
    <> MyThirdCase

-- or

fn = myFirstCase
    <> MySecondCase
    <> MyThirdCase
```

It's purely aesthetic :man_shrugging: